src/osd/OSD.cc

```diff
@@ -5544,6 +5571,7 @@ void OSD::_collect_metadata(map<string,string> *pm)
     // not applicable for bluestore
     (*pm)["osd_journal"] = journal_path;
   }
+
```

Comment:
please avoid adding empty lines.

Reply:
I think it vastly improves readability.

Reply:
see https://github.com/ceph/ceph/blob/master/CodingStyle and https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace. also it'd be better not changing the area your are not touching.